src/site/elements/utils/rester-authorization-provider-oauth2-configuration-dialog.js

```diff
@@ -146,6 +146,7 @@ const template = html`
                             <paper-item value="none"
                                 >None (public client)</paper-item
                             >
+                            <paper-item value="pkce">PKCE</paper-item>
```

Comment:
Is PKCE only possible for code flow? Might need to hide that option for other flows.